components/survey/personal-information.tsx

```diff
@@ -1,15 +1,237 @@
-import { View, Text } from "react-native";
-import React from "react";
-import Header from "../common/Header";
+import React, { useMemo, useState, useEffect } from "react";
+import {
+  View,
+  Text,
+  TextInput,
+  Pressable,
+  Platform,
+  ScrollView,
+  KeyboardAvoidingView,
+} from "react-native";
+import Header from "@/components/common/Header";
+import { RadioButtonProps } from "react-native-radio-buttons-group";
+import DateTimePicker, {
+  DateTimePickerEvent,
+} from "@react-native-community/datetimepicker";
+import * as Location from "expo-location";
 
 const PersonalInformation = () => {
+  const [selectedId, setSelectedId] = useState<string | undefined>();
+  const [birthday, setBirthday] = useState("");
+  const [date, setDate] = useState<Date>(new Date());
+  const [show, setShow] = useState(false);
+  const [location, setLocation] = useState<Location.LocationObject | null>(
+    null
+  );
+  const [errorMsg, setErrorMsg] = useState<string | null>(null);
+  const [address, setAddress] = useState<string | null>(null);
+
+  const gender: RadioButtonProps[] = useMemo(
```

Comment:
I think di na naton need mag useMemo diri yan, separate nalang to a different file sa `constants` folder to something like `genders.ts`